versions/3.1.0.md

```diff
@@ -2336,6 +2336,20 @@ As such, inline schema definitions, which do not have a given id, *cannot* be us
 The [xml](#schemaXml) property allows extra definitions when translating the JSON definition to XML.
 The [XML Object](#xmlObject) contains additional information about the available options.
 
+###### Picking Schema Vocabularies
+
+OpenAPI documents are made up of multiple resources. Sometimes resources are all in the same root OpenAPI document (inline resources); also, resources can contain their own documents with `$ref` (separate resources). A schema object could exist as an inline resource, or a separate resource. 
+
+It is important for tooling to be able to detect what meta-schema any given resource wishes to be processed with: JSON Schema Core, JSON Schema Validation, OpenAPI Schema Object, or some custom meta schema.
+
+The `$schema` keyword can be used to identify which meta-schema should be used when interacting with any schema resource.
+
+Inline schema resources will default to using `$schema: "https://spec.openapis.org/oas/3.1/schema-object"`.
```

Comment:
```suggestion
Inline schema resources MUST default to using `$schema: "https://spec.openapis.org/oas/3.1/schema-object"`.
```

Reply:
Pending resolution of the outstanding question of whether these defaults are desirable (allowing for future versions of JSON Schema) or implicit/explicitly stated in some new top-level property/object.